components/app/feed/ProjectPostSkeletonCard.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="shadow bg-base-200 rounded p-8 h-275px">
```

Comment:
suggestion(non-blocking): i think we do not need this outer wrapper

Reply:
Added this to the dedicated ticket. See https://github.com/ocean-tech-ship/ocean-pearl/issues/759#issuecomment-1264626844